src/Microsoft.AspNet.Antiforgery/DefaultAntiforgery.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNet.Antiforgery
     /// Provides access to the anti-forgery system, which provides protection against
     /// Cross-site Request Forgery (XSRF, also called CSRF) attacks.
     /// </summary>
-    public class Antiforgery
+    public class DefaultAntiforgery : IAntiforgery
```

Comment:
Thought we were moving away from `DefaultBlah` names? That is, why not leave this as `AntiForgery`?